src/include/gurt/telemetry_common.h

```diff
@@ -155,6 +155,9 @@ enum {
 	D_TM_SERVER_PROCESS		= 0x000,
 	D_TM_SERIALIZATION		= 0x001,
 	D_TM_RETAIN_SHMEM		= 0x002,
+	D_TM_RETAIN_SHMEM_IF_NON_EMPTY	= 0x004,
+	D_TM_CLIENT_PROCESS		= 0x008,
```

Comment:
I'm wondering why the metrics library needs to know it's a client process at all.

Reply:
oh, yes, this can be removed. thanks.